e2e/support/helpers/e2e-collection-helpers.ts

```diff
@@ -5,12 +5,43 @@ import {
   getFullName,
   popover,
 } from "e2e/support/helpers";
+import type { Collection, CollectionId } from "metabase-types/api";
+
+export const createCollection = ({
+  name,
+  description = null,
+  parent_id = null,
+  authority_level = null,
+}: {
+  name: string;
+  description?: string | null;
+  parent_id?: CollectionId | null;
+  authority_level?: "official" | null;
+}): Cypress.Chainable<Cypress.Response<Collection>> => {
+  cy.log(`Create a collection: ${name}`);
+
+  return cy.request("POST", "/api/collection", {
+    name,
+    description,
+    parent_id,
+    authority_level,
+  });
+};
+
+export const archiveCollection = (id: CollectionId) => {
+  cy.log(`Archiving a collection with id: ${id}`);
+
+  return cy.request("PUT", `/api/collection/${id}`, {
+    archived: true,
+  });
+};
 
 /**
  * Clicks the "+" icon on the collection page and selects one of the menu options
- * @param {"question" | "dashboard" | "collection"} type
  */
-export function openNewCollectionItemFlowFor(type) {
+export function openNewCollectionItemFlowFor(
+  type: "question" | "dashboard" | "collection",
```

Comment:
Should we add models and metrics to the list?

Reply:
It's only ever used for "collections" at the moment.
Actually `newButton` helper should be used instead.

So I deprecated this function and I'm leaving the implementation as is: eb9a98020ae59a49a4d7388944025b4aad353903